CHANGES_NEXT_RELEASE

```diff
@@ -1,12 +1 @@
-- Add: notification metadata previousValue and actionType (except 'delete') (#2507)
-- Add: filtering metadata in notifications (#2507)
-- Fix: removed spurious decimals in metadata numbers in NGSIv1 rendering (a follow up of #2176)
-- Fix: wrong attributes included in notifications at subscription update time in some cases (both in blacklist and not blacklist cases)
-- Fix: csub cache wrong deletion of subscription at subscription update time when new subscription has empty condition attributes list
-- Fix: crash when creating entity with metadata when subscription is in place with "mq" on a different metadata (#2496)
-- Fix: correct rendering of metadata compound values when a vector (bugfix for issue #1068)
-- Fix: crash with request with empty URI PATH (Issue #2527)
-- Fix: supporting DateTime filters for metadata (#2443, #2445)
-- Fix: wrong matching in metadata existence and not existence filters with compounds
-- Fix: missing expression sub-fields (q, etc.) in csubs makes Orion to crash
-- Hardening: ensure to use type oriented functions when getting data from DB (safer, but more verbose in error log if DB is not perfectly aligned with what Orion is expecting)
+- Add: new value for URI param 'options' (options=noAttrDetail) for lighter queries (Issue #2073) 
```

Comment:
I'd suggest to be more precise:

```
Add: new value for URI param 'options' (options=noAttrDetail) for NGSIv2 type browsing queries to avoid attribute detail, making the query faster (#2073) 
```

or something like that. With regards to NGSIv1, this PR also involves a fix (in a second CNR line):

```
Fix: avoid unused attribute type detail in NGSIv1 type browsing queries (which was making queries slower) (#2073)
```

or something like that (probably in this case my wording proposal is not too good... but I guess you get the idea).

Finally, I'll suggest to remove "Issue" before the number. As every #XXXX refers to an issue number, "Issue" is redundant.